CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased][]
 
+### Added
+
+-   `idoit-install`: Add support for CentOS 8.1
```

Comment:
8.1 only? :thinking: I think you mean the hole 8 branch, right?

Reply:
I tested version 8 and the actual version 8.1. Therefore I accidentally wrote the latest version. I tested both again...just to be safe ;) 
Now I adjusted the changelog (again) for the whole 8 branch.